### ISSUE TITLE
conv2d function is not defined 'batch_norm' keyword parameter

### ISSUE THREAD

**Issue by stozpark** [pkmital/tensorflow_tutorials#9]
10_residual_network.py

has some problem.

conv2d function is not defined 'batch_norm' keyword parameter.

**Comment by pkmital** [comment]
Yes sorry I recently changed the implementation of batch_norm.  This change no longer allowed me to place it within the conv2d function as a simple branch, as it requires an additional placeholder for training/not training.  It can be added to the residual network by applying batch norm then the activation.  For now, I've simply removed batch norm from the residual network implementation so that it runs, but it is a really bad idea to try and train anything that deep without it.
